pkg/config/env/mongo.go

```diff
@@ -67,3 +74,52 @@ func MongoConnectionTimeoutWithDefault(value string) string {
 func MongoSSLWithDefault(value string) string {
 	return envy.Get("MONGO_SSL", value)
 }
+
+// MongoConnDetails fetches appropriate environment variables from other
+// Mongo* functions and returns a mongo.ConnDetails according to their
+// values. Otherwise, returns nil and a non-nil error
+func MongoConnDetails() (*mongo.ConnDetails, error) {
+	host, err := MongoHost()
+	if err != nil {
+		return nil, err
+	}
+	portStr, err := MongoPort()
+	if err != nil {
+		return nil, err
+	}
+	port, err := strconv.Atoi(portStr)
+	if err != nil {
+		return nil, err
+	}
+
+	user, err := MongoUser()
+	if err != nil {
+		return nil, err
+	}
+
+	password, err := MongoPassword()
+	if err != nil {
+		return nil, err
+	}
+
+	timeoutSecStr := MongoConnectionTimeoutWithDefault(defaultMongoTimeoutSec)
```

Comment:
can we reuse https://github.com/gomods/athens/blob/514ac4965992b165cd6a63fde71a2ba29c309d24/pkg/config/env/timeout.go#L11
or does it need to be different one

Reply:
I think it should be one specific to mongo. going forward we should probably have other specialize timeouts with defaults like this one. the `Timeout()` function seems a bit generic